tests/ordering/tests.py

```diff
@@ -462,6 +462,12 @@ def test_default_ordering_by_f_expression(self):
             attrgetter('headline')
         )
 
+    def test_default_parent_ordering_by_expression(self):
+        sub1 = SubArticle.objects.create(headline='Sub 1', pub_date=datetime(2005, 7, 26))
+        sub2 = SubArticle.objects.create(headline='Sub 2', pub_date=datetime(2005, 7, 27))
+        articles = SubArticle.objects.order_by('article_ptr')
+        self.assertSequenceEqual(articles, [sub2, sub1])
```

Comment:
We should add third `SubArtictle` to check if expressions are used in ordering, e.g.
```diff
diff --git a/tests/ordering/tests.py b/tests/ordering/tests.py
index 39b5a75564..886c97e60f 100644
--- a/tests/ordering/tests.py
+++ b/tests/ordering/tests.py
@@ -464,9 +464,10 @@ class OrderingTests(TestCase):
 
     def test_default_parent_ordering_by_expression(self):
         sub1 = SubArticle.objects.create(headline='Sub 1', pub_date=datetime(2005, 7, 26))
-        sub2 = SubArticle.objects.create(headline='Sub 2', pub_date=datetime(2005, 7, 27))
+        sub2 = SubArticle.objects.create(headline='Sub 2', pub_date=datetime(2005, 7, 27), author=self.author_2)
+        sub3 = SubArticle.objects.create(headline='Sub 2', pub_date=datetime(2005, 7, 27), author=self.author_1)
         articles = SubArticle.objects.order_by('article_ptr')
-        self.assertSequenceEqual(articles, [sub2, sub1])
+        self.assertSequenceEqual(articles, [sub3, sub2, sub1])
```